site/components/src/main/java/com/visitscotland/brxm/services/ResourceBundleService.java

```diff
@@ -54,16 +54,13 @@ public String getResourceBundle(String bundleName, String key, Locale locale){
     }
 
     public String getSiteResourceBundle(String bundleName, String key, Locale locale){
-
-        //Note: Other sites don't currently support other languages
-        if (!properties.getSiteId().isEmpty()) {
-            String siteBundleName = properties.getSiteId() + "." + bundleName;
-            // TODO: This method throws an exception when the site bundle is not found. This bug should be fixed
-            if (getResourceBundleRegistry().getBundle(siteBundleName) != null) {
-                String value = getResourceBundle(siteBundleName, key, locale, true);
-                if (value != null) {
-                    return value;
-                }
+        if (!properties.getSiteId().isEmpty() && hasSiteResourceBundle(bundleName, locale)) {
```

Comment:
I think `isEmpty()` is based from a length check and does not account for whitespace, could `isBlank()` be more accurate?

```suggestion
        if (!properties.getSiteId().isBlank() && hasSiteResourceBundle(bundleName, locale)) {
```

Reply:
There is no such risk with this field as all the values are trimmed. But I'll accept the changes as a practice/intention